tools/sotn-assets/assets/cutscene/handler.go

```diff
@@ -212,6 +223,9 @@ func parseScript(r io.ReadSeeker, baseAddr, addr psx.Addr, length int) ([][]stri
 			script = append(script, command)
 		} else if op < 0x7F {
 			text += string([]byte{byte(op)})
+		} else if os.Getenv("VERSION") == "pspeu" && op >= 0xA0 && op <= 0xDF {
```

Comment:
I am not sure if it is necessary but in the interest of defensive coding I only made this happen for PSP, since PSX does not use multi-lang characters

Reply:
Here the same previous comment applies, where we could just always dump the hex literal if it's easier. If you wish to still output a character and let `sotn_str` handle the rest, we should move `os.Getenv` out of each loop. The main reason being the use of a mutex in the `os.Getenv` to ensure multi-thread synchronization, which will impact the performance of building cutscenes on a CPU that allows great concurrency.

Reply:
I will move the version check outside the loop, thanks for the hint, I didn't know that about the performance characteristics.

There are still some BYTE data which I don't think are actually part of strings which I think we would probably like to preserve, rather than always treating these higher order characters as string values, so I think this check probably makes sense (at least for now)

eg.
![image](https://github.com/user-attachments/assets/54b106e4-3af6-43bd-b472-5fbbdfd98266)

Reply:
Done 👍